lib/python/pyflyby/_autoimp.py

```diff
@@ -558,9 +562,17 @@ def visit_Lambda(self, node):
                 self.visit(node.body)
             self._in_FunctionDef = old_in_FunctionDef
 
+    def _visit_typecomment(self, typecomment):
+        if typecomment is None:
+            return
+        node = ast.parse(typecomment)
```

Comment:
As a note, i'm unsure if this line was ever hit, as I can't get `ast.parse()` to parse a type comment on it's own like that in python 3.8 or above, I have to pass "type_comments=True", and even then it always fails.